bin/remixd

```diff
@@ -65,6 +67,16 @@ if (program.profiler) {
   killCallBack.push(profilerRouter.start())
 }
 
+if (program.truffle) {
+  var truffleRouter = new Router(65522, servicesList['truffle'])
+  killCallBack.push(truffleRouter.start())
+}
+
+if (program.npm) {
```

Comment:
not sure that 's really needed, why do you need to have npm CLI has a service?

Reply:
It would be extremely useful for javascript testing. I install and save npm dependencies for projects to use npm packages within my test suites. The ability to do this would also make writing migrations and scripts easier from the browser (although the truffle service would need to expand to accommodate those features).